### ISSUE TITLE
Missing: remove favorite button

### ISSUE THREAD

**Issue by siamkreative** [SiamKreative/ThaiFood.today#18]
There should be a button (`<i class="fa fa-trash" aria-hidden="true"></i>`) to remove a dish from your favorites.
